### DISCUSSION TITLE
Additional text box in upper LH corner?

### DISCUSSION THREAD

**Discussion by aumusic** [dialogic-godot/dialogic#1781] [Q&A]
Has anybody else ever come across an additional, very small text box appearing in the upper LH corner of their program window when running Godot (4.1) program tests VIA the "run program" button in the RH corner? 

![quicktest](https://github.com/coppolaemilio/dialogic/assets/52223818/4b41549c-08c3-48de-9b72-629709e3c6a4)

Note that I don't get this additional text box when running Dialogic tests VIA the "play timeline" button within the Dialogic window, only when using Godot's "run program" button. The code that I'm using to have Godot start Dialogic playback is terribly simple (perhaps too simple), so I'm not sure what is causing this behavior:

`extends Node

func _ready():
	Dialogic.start('testno1.dtl')`

Has anyone come across this before?

Thanks in advance

**Comment by Jowan-Spooner** [comment]
I've not noticed this before. Have you setup a custom style with a custom layout scene?

**Comment by aumusic** [comment]
I just did some testing in a new project by customizing the same Dialogic variables as I had in the previous project, and I have not been able to re-create it. My guess right now is that this isn't a Dialogic problem at all and that I've screwed up elsewhere, but I'll do some more testing.
